test-suit/starryos/normal/test-mt-execve/c/src/main.c

```
@@ -0,0 +1,130 @@
+}
+
+static void *sibling_block(void *arg)
+{
+    (void)arg;
+    pthread_mutex_lock(&mtx);
+    sibling_ready++;
+    pthread_cond_broadcast(&cond);
+    pthread_mutex_unlock(&mtx);
+
+    /* Block until the exec path SIGKILLs us. */
+    for (;;) pause();
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)